src/regfox/regfox_api.js

```diff
@@ -79,15 +81,45 @@ const login = async (email, password) => {
   }).then(response => response.json())
     .then(response => {
       if (get(response, 'data.mutationResponse.errors')) {
-        throw new Error(response);
+        throw new Error(`Could not parse or process response, see details to fix here: ${JSON.stringify(response)}`);
       }
 
       if (!get(response, 'data.mutationResponse.success', false)) {
-        throw new Error(response);
+        throw new Error(`Could not parse or process response, see details to fix here: ${JSON.stringify(response)}`);
       }
 
       return response.data.mutationResponse;
     });
 }
 
-export { searchRegistrations, exchangeBearerToken, login, REGFOX_GRAPHQL_URL, REGFOX_EXCHANGE_TOKEN_URL };
\ No newline at end of file
+/**
+ * Returns a nicely parsed object containing registration info given an id.
+ * Returns a promise Error if the network is down or parsing failed.
+ * 
+ * @param {*} id of a registration, note this is **not** the RegistrationId, its just the id
```

Comment:
Should this be a string?

```suggestion
 * @param {string} id of a registration, note this is **not** the RegistrationId, its just the id
```

Reply:
You and your...types. 👍